src/config/bindings.rs

```diff
@@ -199,9 +189,18 @@ pub fn platform_key_bindings() -> Vec<KeyBinding> {
         Key::Add, [ctrl: true]; Action::IncreaseFontSize;
         Key::Subtract, [ctrl: true]; Action::DecreaseFontSize;
         Key::Minus, [ctrl: true]; Action::DecreaseFontSize;
-    ));
+    );
 
-    bindings
+    if cfg!(all(target_os = "windows", not(test))) {
```

Comment:
We shouldn't use `cfg!`, since this is a runtime check instead of a compile-time check.

I'd also moving this into the `platform_key_bindings (using a dedicated one for Windows), these were created exactly for the issue of test requiring no platform-specific bindings.

To prevent us from having to duplicate the windows and linux bindings since they're largely similar, we could put those in a separate method that is only offered on linux/bsd/windows and called from the more specific platform_bindings.